### DISCUSSION TITLE
Обновление Zapret на OpenWRT

### DISCUSSION THREAD

**Discussion by pavelmoshik** [bol-van/zapret#867] [Q&A]
Здравствуйте. Подскажите как обновить zapret на OpenWRT без переустановки?

**Comment by numbereleven-a** [comment]
как и установка через install_easy.sh и дальше отвечать на вопросы

**Comment by truechupa** [comment]
Если версия старая, то лучше с полной переустановкой

**Comment by bol-van** [comment]
Если там старый конфиг с HTTP,HTTPS,QUIC то инстяллятор предупредит и откажется сохранять конфиг.
Можно согласиться с полным сносом, тогда дело пойдет дальше

**Comment by pavelmoshik** [comment]
> Если там старый конфиг с HTTP,HTTPS,QUIC то инстяллятор предупредит и откажется сохранять конфиг. Можно согласиться с полным сносом, тогда дело пойдет дальше

Если я запускаю install_bin.sh, то появляется: 

> root@OpenWrt:~# ./install_bin.sh
> no compatible binaries found


После запуска install_easy.sh:

> root@OpenWrt:~# ./install_easy.sh
> * checking system
> system is based on openwrt
> package manager opkg
> firewall fw4. flow offloading requires nftables.
> * checking executables
> find: /root/binaries: No such file or directory
> build tools not found
> 
> press enter to continue

Все зависимости и пакеты установлены и обновлены.

**Comment by bol-van** [comment]
Релиз надо брать. Из репы бинари убраны

**Comment by ptipti23** [comment]
> Релиз надо брать. Из репы бинари убраны

А как это будет звучать, если перевести на русский?

**Comment by NOFEXtreme** [comment]
> А как это будет звучать, если перевести на русский?

Скачать релиз: https://github.com/bol-van/zapret/releases
Почитать как установить: https://github.com/bol-van/zapret/blob/master/docs/quick_start.md#%D0%BD%D0%B0%D1%81%D1%82%D1%80%D0%BE%D0%B9%D0%BA%D0%B0

**Comment by lux01sv** [comment]
Немного не в тему, знаю, будет релиз в формате .apk пакета для новых версий openwrt ?

**Comment by bol-van** [comment]
Пакетированием не буду заниматься точно.
Если кто-то и будет делать такое, то только отдельно

**Comment by remittor** [comment]
> Немного не в тему, знаю, будет релиз в формате .apk пакета для новых версий openwrt ?

Ждите apk на http://github.com/remittor/zapret-openwrt

Появятся как только пойму в чём трабла сборки....
